lib/lifecycles/bump.js

```diff
@@ -222,10 +222,16 @@ function updateConfigs(args, newVersion) {
     }
     const configPath = path.resolve(process.cwd(), updater.filename);
     try {
-      if (dotgit.ignore(updater.filename)) return;
+      if (dotgit.ignore(updater.filename)) {
+        console.debug(updater.filename + ' is ignored in Git')
+        return;
+      }
       const stat = fs.lstatSync(configPath);
 
-      if (!stat.isFile()) return;
+      if (!stat.isFile()) {
+        console.debug(updater.filename + ' is not a file')
```

Comment:
```suggestion
        console.debug(`Not updating '${updater.filename}', as it is not a file`)
```

^ Similar suggestion to above